modules/lsm/lsm_module_iface.c

```diff
@@ -166,6 +177,44 @@ _drive_update (UDisksObject   *object,
     }
 }
 
+static gboolean
+_lsm_local_check (UDisksObject *object)
+{
+  /* LsmLocalDisk interface will be available on all disk drive as
+   * we don't have reliable way to tell whether LED control works or not
+   * forehead, user have to try and handle errors.
```

Comment:
It's SCSI SES standard we should blame not HW vendor:

SES-3 rev14 quote:

An IDENT bit set to zero indicates that the enclosure services process is not identifying the element by a visual indicator as a result of the RQST IDENT bit in the Device Slot control element __or a visual indicator is not implemented__.